terraform/modules/iam_role/riemann_monitoring/policy.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:DescribeLogGroups",
+        "logs:DescribeLogStreams"
+      ],
+      "Resource": [
+        "arn:${var.aws_partition}:logs:${var.aws_default_region}:${var.account_id}:*"
```

Comment:
For some reason, we don't need the `var.` prefix in templates: https://www.terraform.io/docs/providers/template/d/file.html